### DISCUSSION TITLE
Unable to generate shared library file for ios

### DISCUSSION THREAD

**Discussion by xengine-qyt** [openssl/openssl#22196] [Q&A] [answered]
./Configure iphoneos-cross shared ,

only static .a file..how to generate shared file for ios?

**Comment by levitte** [comment, marked as answer]
Our config template for all the ios configs disable shared library building:

https://github.com/openssl/openssl/blob/91bc783a93a2a695fe6a2f8da93cf5b5e086ba42/Configurations/15-ios.conf#L10-L15

That happened in this commit, and the commit message explains the reasoning: aa58071e4b8b245db1564f476731c978738e7e98
Feel free to read it.  Oh, and if things have changed on iOS, an update PR is certainly welcome

**Reply by xengine-qyt** [reply]
o,thanks
